test/inactive-blog-filter.test.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs');
+const inactiveFilter = require('../src/inactive-blog-filter');
+
+const redlist = JSON.parse(fs.readFileSync('feeds-redlist.json', 'utf8'));
+
+describe('Redlisted feed checking', () => {
+  it('should return false for bad feed URLs', async () => {
```

Comment:
You can remove `async` here, everything you're doing is sync

Reply:
(Resolved by b8019ac.)